drivers/jwt/Kconfig

```diff
@@ -0,0 +1,31 @@
+# This is a hack.
+
+menuconfig JWT
+	bool
+	prompt "Enable JWT token generation"
+	select JSON_LIBRARY
+	default n
+	help
+	  Enable creation of JWT tokens
```

Comment:
Could make this

```
menuconfig JWT
	bool "Enable JWT token generation"
	select JSON_LIBRARY
	help
	  Enable creation of JWT tokens
```

`bool` symbols implicitly default to `n`, and `bool "Foo"` is a shorthand for`bool` + `prompt "Foo"`.